README.md

```diff
@@ -1,7 +1,9 @@
 # AliSQL
+
 AliSQL is a MySQL branch originated from Alibaba Group. It is based on the MySQL official release and has many feature and performance enhancements. AliSQL has proven to be very stable and efficient in production environment. It can be used as a free, fully compatible, enhanced and open source drop-in replacement for MySQL.
 
-AliSQL has been an open source project since August 2016. It is being actively developed by engineers from Alibaba Group. Moreover, it includes patches from Percona, WebScaleSQL, and MariaDB. AliSQL is a fruit of community effort. Everyone is welcomed to get involved.
+**AliSQL has been discontinued, please use [ApsaraDB/galaxyengine](https://github.com/ApsaraDB/galaxyengine) instead.**
```

Comment:
galaxyengine and alisql are not maintained by one team, and is also not actively maintained, so how about using stonedb instead?

```suggestion
**AliSQL has been discontinued, please use [stoneatom/stonedb](https://github.com/stoneatom/stonedb) instead.**
```

Reply:
@xpchild Add a link `Repository AliSQL 8.0` to https://github.com/ApsaraDB/galaxyengine in commit https://github.com/alibaba/AliSQL/commit/25a2aefb4329a62db747e74611888cbb4fff87a4, so I use it directly.

I do not know about its internal maintenance relation. Any repo LGTM if maintainer approved.